README.md

```diff
@@ -35,38 +35,49 @@ go get github.com/NodeFactoryIo/vedran
 ```
 Note that if you need to do this, you probably want to add your Go bin directory to your $PATH to make things easier!
 
-## Usage
+## Starting loadbalancer
 
-Load balancer is started using command `start`.
+First download latest prebuilt binaries [from releases](https://github.com/NodeFactoryIo/vedran/releases).
 
-```
-$ vedran -h
+Load balancer is started by invoking **start** command.
```

Comment:
add note that for more commands and options run `vedran --help` or `vedran start --help`

Reply:
:+1: